Mathlib/Order/Filter/Basic.lean

```diff
@@ -180,6 +180,18 @@ theorem congr_sets (h : { x | x ∈ s ↔ x ∈ t } ∈ f) : s ∈ f ↔ t ∈ f
     mp_mem hs (mem_of_superset h fun _ => Iff.mpr)⟩
 #align filter.congr_sets Filter.congr_sets
 
+/-- Override `sets` field of a filter to provide better definitional equality. -/
+protected def copy (f : Filter α) (S : Set (Set α)) (hmem : ∀ s, s ∈ f ↔ s ∈ S) : Filter α where
```

Comment:
nit:
```suggestion
protected def copy (f : Filter α) (S : Set (Set α)) (hmem : ∀ s, s ∈ s ↔ s ∈ f) : Filter α where
```
matches the order of `Submonoid.copy`, but maybe we're just inconsistent.